### PR TITLE
awoa 9

### PR DESCRIPTION
**Pull request by ngokevin**
screen shot 2016-04-14 at 10 00 38 pm

### CODE REVIEW

**Comment by ngokevin** [comment]
@cvan r? gonna post this in a bit

**Comment by ngokevin** [comment]
lmk if you catch anything

File: scripts/helpers.js [hunk 1]
```diff
@@ -71,6 +71,28 @@ hexo.extend.helper.register('blog_date_subtract_week', function (date) {
...
+/**
+ * #1000 -> github.com/aframevr/aframe/pull/1000
+ * abcde -> github.com/aframevr/aframe/commit/abcde
+ * aframevr/aframe-editor|#1000 -> github.com/aframevr/aframe-editor/pull/1000
+ */
+hexo.extend.helper.register('github_contribution', function (contribution, display) {
+  var project = 'aframevr/aframe';
+  display = display || contribution;
+  console.log(contribution);
+  if (contribution.indexOf('|') !== -1) {
```
Comment (cvan) [inline scripts/helpers.js]: no need to check

File: scripts/helpers.js [hunk 2]
```diff
@@ -71,6 +71,28 @@ hexo.extend.helper.register('blog_date_subtract_week', function (date) {
   return moment(date).subtract({weeks: 1}).format('MMM D[,] YYYY');
 });
 
+/**
+ * #1000 -> github.com/aframevr/aframe/pull/1000
+ * abcde -> github.com/aframevr/aframe/commit/abcde
+ * aframevr/aframe-editor|#1000 -> github.com/aframevr/aframe-editor/pull/1000
+ */
+hexo.extend.helper.register('github_contribution', function (contribution, display) {
+  var project = 'aframevr/aframe';
```
Comment (cvan) [inline scripts/helpers.js]: `var project = this.config.github.aframe.username + '/' + this.config.github.aframe.repo;`

File: scripts/helpers.js [hunk 3]
```diff
@@ -71,6 +71,28 @@ hexo.extend.helper.register('blog_date_subtract_week', function (date) {
...
+  display = display || contribution;
+  console.log(contribution);
```
Comment (cvan) [inline scripts/helpers.js]: remove?

**Pull request merged**